config/configuration.toml.in

```diff
@@ -20,6 +20,24 @@ default_vcpus = -1
 #default_memory = @DEFMEMSZ@
 disable_block_device_use = @DEFDISABLEBLOCK@
 
+# Enable pre allocation of VM RAM, default false
+# Enabling this will result in lower container density
+# as all of the memory will be allocated and locked
+# This is useful when you want to reserve all the memory
+# upfront or in the cases where you want memory latencies
+# to be very predictable
+# Default false
+#enable_mem_prealloc = true
+#
```

Comment:
Replace these `# ` blank lines with just blank lines to keep in the style of the rest of the file.
Although, the reset of the file is not wholly consistent, I think breaking up these sections will fully blank lines will make it clearer.